queries/elixir.scm

```diff
@@ -0,0 +1,41 @@
+(comment) @comment
+(anonymous_function) @anonymousFunction
+(arguments) @arguments_list
+(call
+  target: (_) @functionCallee) @functionCall @functionCallee.domain
+
+;;(keywords
+;;  (pair
+;;    key: (_) @collectionKey @collectionItem.trailing.end.startOf
+;;    value: (_) @value) @collectionItem @collectionItem.trailing.start.endOf
+;;    (",")? @collectionItem.trailing.start.startOf
+;;    (#allow-multiple! @collectionItem))
+
+;; How does surrounding whitespace work for this
+(map_content
+  (binary_operator
+      left: (_) @collectionKey @collectionKey.trailing.start.endOf
+      right: (_) @value @collectionKey.trailing.end.startOf) @collectionItem
+      ","? @collectionItem.trailing
+      (#shrink-to-match! @collectionItem "([^,]+)"))
+
+(call
+  target: (identifier) @ignore
```

Comment:
We tend to use the term `dummy` for things that aren't scope types, and are just used for predicates. At some point that may be enforced

Reply:
although see https://github.com/cursorless-dev/cursorless/pull/1839#discussion_r1314817213

Reply:
I've switched to `_target`, also I liked putting the predicate immediately after the match

Reply:
Cool. As you can see, this stuff is fairly new for us, so we're still finding the conventions that we like 😊